articles/automation-testing-in-selenium/index.md

```diff
@@ -0,0 +1,99 @@
+
+Web driver is faster than RC and does not require server dependency to conduct test automation. This explains why it is widely used compared to RC.  
+
+### Types of Testing That Can be Done Using Selenium Web Driver
+
+* **Unit testing:** This involves the testing of specific source code units. Some of the aspects tested in this category include program modules, operating procedures, usage procedures, and control data. 
+* **System testing:** This automation testing examines whether the system complies with certain requirements that have been set. 
+* **Integration testing:** In this category, testers examine whether the program modules work well even after integration. 
+* **End-to-end testing:** This involves an evaluation of the whole software, including the external surfaces. It also evaluates how the system communicates and integrates with other applications, networks, interfaces, and databases. 
+* **Regression testing:** This testing ensures that new codes do not affect the functionality of the existing web application. It enhances a seamless addition of new features.
+* **Automated browser testing:** Automated testing ensures that the developed web application can be used in various browsers without experiencing problems. 
+* **Performance and Load Testing:** This is used for testing the performance of web applications and their load time across various browsers.
+* **Monkey testing:** This is a random test in which random inputs are used to check whether the application works well. 
+
+### Limitations of Selenium Automation Testing
+* It cannot be used for testing mobile applications.
+* Test management can only be done through integration with other applications. This is because it is not supported by the Selenium framework.
+* Using Selenium for automation testing requires users to be conversant with programming languages. 
+* It does not support the testing of images.
+
+### Conclusion
+Selenium is a cost-effective and flexible tool that can be used by developers in the automation testing of their web applications. The most intriguing feature of this software is the ability to test applications across various web browsers. This ensures that websites do not crush or break down in some browsers. 
+
+Selenium software is ideal for companies developing applications that support heavy traffic, especially social platforms and e-commerce websites. This software has undergone improvement over the years, which has added value to web development. 
+
+### Resources
+
+[Selenium Expertz](https://seleniumexpertz.wordpress.com/2020/03/16/introduction-to-selenium/)
+
+[Selenium Easy](https://www.seleniumeasy.com/selenium-tutorials/introduction-to-selenium)
+

```

Comment:
It would add some value to this article if you could add a brief guide on setting up Selenium Web Driver and setting up some tests. Or you could also write a part 2 for this article where you discuss in detail.

Reply:
Thank you for this suggestion. I will write another article where this will be discussed in detail.